dbschema/seeds/017.partnerships.edgeql

```diff
@@ -0,0 +1,83 @@
+with
+partnershipsJson := to_json($$[
+  {
+      "partner": "Rohan Linguistics",
+      "project": "South Downs",
+      "mouStatus": "NotAttached",
```

Comment:
NotAttached is default for mouStatus and agreementStatus, and really I don't think it's used much. Let's just remove from these seeds.